core/deployment/src/main/java/io/quarkus/deployment/jbang/JBangAugmentorImpl.java

```diff
@@ -0,0 +1,156 @@
+package io.quarkus.deployment.jbang;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.*;
```

Comment:
Avoid star imports

Reply:
yeah, I don't use them but recently the formatter seem to make them happen.